amazon-kinesis-client/src/main/java/software/amazon/kinesis/metrics/MetricsUtil.java

```diff
@@ -19,13 +19,15 @@
 
 import lombok.NonNull;
 import software.amazon.awssdk.services.cloudwatch.model.StandardUnit;
+import software.amazon.kinesis.common.StreamIdentifier;
 
 /**
  *
  */
 public class MetricsUtil {
```

Comment:
Let's add metrics for the number of streams being processed by the KCL when it is greater than 1.